e2e/atls/atls_test.go

```diff
@@ -54,7 +54,7 @@ func TestSNPValidators(t *testing.T) {
 
 	coordinator = kuberesource.AddPortForwarders(coordinator)
 
-	ct.Init(t, coordinator)
+	ct.Init(t, coordinator, false)
```

Comment:
passing true/false/nil to a function makes the code hard to read on the caller side. You could define aliases in the package defining the function that could be used instead.